dockstore-webservice/src/main/java/io/dockstore/webservice/core/ChecksumConverter.java

```diff
@@ -0,0 +1,50 @@
+package io.dockstore.webservice.core;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Converter(autoApply = true)
+public class ChecksumConverter implements AttributeConverter<List<Checksum>, String> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ChecksumConverter.class);
+    @Override
+    public String convertToDatabaseColumn(List<Checksum> checksums) {
+        String stringChecksums;
+        if (checksums != null && !checksums.isEmpty()) {
+            try {
+                stringChecksums = checksums.stream().map(checksum -> checksum.getType().trim() + ":" + checksum.getChecksum().trim()).collect(Collectors.joining(","));
+            } catch (NullPointerException ex) {
+                LOG.error("Could not convert checksum(s) to string for database", ex);
+                return null;
+            }
+            return stringChecksums;
+        }
+        return null;
+    }
+
+    @Override
+    public List<Checksum> convertToEntityAttribute(String checksumString) {
```

Comment:
It would be helpful to have JavaDoc either here for the class explaining the format, perhaps with an example. I can construct it by reading the code but it would be easier to understand if you had in the comments that the format in the database is `type:checksum`